charts/cosigned/templates/webhook/service_monitor_webhook.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and .Values.serviceMonitor.enabled (.Capabilities.APIVersions.Has "monitoring.coreos.com/v1") }}
```

Comment:
I would generally avoid `Capabilities` and `lookup` in helm charts. Some tools like ArgoCD, helm diff ( https://github.com/databus23/helm-diff/issues/165 ) use `helm template` instead of `helm install/upgrade`. `helm template` has no connection to the Kubernetes API server and therefor the condition will never be true.
I would let the apply of the ServiceMonitor fail if the resource is unknown to the cluster.
The feature flag `serviceMonitor.enabled` should be enough.

Reply:
agree with this comment. the lack of support through GitOps tools, like argocd

Reply:
no strong opinions but makes sense 
+1